### DISCUSSION TITLE
docs/ninjutsu/shadow-ninja/get-more-money/

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#13] [General]
# docs/ninjutsu/shadow-ninja/get-more-money/

不管怎么摸鱼，都只是白嫖老板，但是摸鱼人肯定不会就这么停下，虽然打工人的肉身还在工位上，但他们的元神早就不知道跑到哪里去了。

https://996.ninja/docs/ninjutsu/shadow-ninja/get-more-money/

**Comment by BaiXiaoZhi** [comment]
此番心法，当真让人受益匪浅

**Comment by WANGPDDD** [comment]
牛逼
